docs/src/main/asciidoc/tika-guide.adoc

```diff
@@ -126,3 +94,141 @@ public class TikaParserResource {
 }
 ----
 
+== Creating the Maven Project
```

Comment:
Could you try to have your doc structured as the others? Typically https://github.com/quarkusio/quarkus/blob/master/docs/src/main/asciidoc/hibernate-search-guide.adoc#solution with the Solution section then the project creation.